pkg/remote/aws/test/sns_topic_policy_multiple/aws_sns_topic_policy-arn_aws_sns_eu-west-3_526954929923_user-updates-topic-arn_aws_sns_eu-west-3_526954929923_user-updates-topic.res.golden.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Typ": null,
+ "Val": null,
+ "Err": "InvalidParameter: Invalid parameter: TopicArn\n\tstatus code: 400, request id: 894d9c65-03ca-501d-bfa5-c4f0fe452458"
```

Comment:
Is this normal ?

Reply:
my bad those were failed golend files that I forgot to delete...